From 768ed58b51412a0b5f26f923d2c523c61104b745 Mon Sep 17 00:00:00 2001
From: Noam Lessner <nlessner@habana.ai>
Date: Tue, 04 Feb 2020 17:57:46 +0200
Subject: [PATCH] [SW-8229] efuse is default location for security

Set efuse as default location for keys, revoke bits and
minimal fw security version.

Change-Id: Id1b89a370f54309687ca3fe7936d89f9805444b4
Signed-off-by: Noam Lessner <nlessner@habana.ai>
---

diff --git a/common/security.c b/common/security.c
index fba177a..7aad862 100644
--- a/common/security.c
+++ b/common/security.c
@@ -527,6 +527,7 @@
 		spi_flash_otp_access(FLASH_OTP_DISABLE);
 		break;
 	case ASIC_EFUSE:
+	default:
 		key_revoke = (arcboot_rom_global.config_fuse &
 			      EFUSE_SYSCFG_REVOKE_KEY_MASK);
 		break;
@@ -555,6 +556,7 @@
 		spi_flash_otp_access(FLASH_OTP_DISABLE);
 		break;
 	case ASIC_EFUSE:
+	default:
 		fw_version_sec = efuse_otp_get_fw_version_security();
 		break;
 	}
@@ -583,6 +585,7 @@
 		spi_flash_otp_access(FLASH_OTP_DISABLE);
 		break;
 	case ASIC_EFUSE:
+	default:
 		efuse_get_hash(key_hash);
 		break;
 	}
